pkg/server/plugin/keymanager/awskms/awskms_test.go

```diff
@@ -0,0 +1,708 @@
+package awskms
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/hashicorp/go-hclog"
+	"github.com/spiffe/spire/pkg/server/plugin/keymanager"
+	"github.com/spiffe/spire/proto/spire/common/plugin"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	// Defaults used for testing
+	validAccessKeyID     = "AKIAIOSFODNN7EXAMPLE"
+	validSecretAccessKey = "secret"
+	validRegion          = "us-west-2"
+	kmsKeyID             = "SPIRE_SERVER_KEY/spireKeyID"
+	spireKeyID           = "spireKeyID"
+)
+
+var (
+	ctx           = context.Background()
+	spireKeyAlias = fmt.Sprintf("%s%s", defaultKeyPrefix, spireKeyID)
+)
+
+func TestKeyManager(t *testing.T) {
+	suite.Run(t, new(KmsPluginSuite))
+}
+
+type KmsPluginSuite struct {
+	// spiretest.Suite
```

Comment:
Is there any particular reason why you are embedding `suite.Suite` rather than `spiretest.Suite`?